src/utils.js

```diff
@@ -4,13 +4,13 @@ const DEPRECATION_MESSAGE = 'isSameUser and isSameDay should be imported from th
 
 export function isSameDay(currentMessage = {}, diffMessage = {}) {
 
+  if(!diffMessage.createdAt) {
+    return false
+  }
+
   let currentCreatedAt = moment(currentMessage.createdAt);
   let diffCreatedAt = moment(diffMessage.createdAt);
 
-  if (!currentCreatedAt.isValid() || !diffCreatedAt.isValid()) {
```

Comment:
I found that `isValid` returns true for all the cases that this test may be trying to avoid. For e.g. `moment(undefined).isValid()`, `moment(0).isValid()`, `moment({}).isValid()` will all return true. I guess one case where it will return false is for `moment(null).isValid()`. I will put the check back in.